packages/design-system/src/icons/svgs/core-24/create-file-24.icon.ts

```diff
@@ -0,0 +1,24 @@
+import {toniqIconColors} from '../../../styles/icon-colors';
+import {html, ToniqSvg} from '../../toniq-svg';
+
+export const CreateFile24Icon = new ToniqSvg(
+    'CreateFile24Icon',
+    html`
+        <svg
+            width="24"
+            height="24"
+            viewBox="0 0 24 24"
+            fill="none"
```

Comment:
The duplicate `fill` attributes below can instead be here to consolidate them:
```suggestion
            fill="${toniqIconColors.stroke()}"
```